lua/nvim-tree/api.lua

```diff
@@ -131,7 +132,7 @@ end
 Api.tree.open = wrap(actions.tree.open.fn)
 Api.tree.focus = Api.tree.open
 
----@class ApiTreeToggleOpts
+---@class ApiTreeToggleOptsApiTreeTo
```

Comment:
What's the reason for this change? `lua/nvim-tree/actions/tree/toggle.lua` is still using this class...

Reply:
No idea 😅 , this was probably done accidentally. Just reverted it 👍🏼